src/components/link/index.tsx

```diff
@@ -0,0 +1,66 @@
+import NextLink from 'next/link'
+import { useId } from '@react-aria/utils'
+import { LinkProps } from './types'
+import { validateProps } from './helpers'
+
+const ARIA_DESCRIBED_BY_PREFIX = 'opens-in-new-tab-label'
+
+/**
+ * A wrapper for the next/link component that also handles unique functionality
+ * with the `opensInNewTab` prop.
+ *
+ * When `opensInNewTab` is passed, a `<span>` with the `g-screen-reader-only`
+ * class is rendered with "(opens in new tab)" as its inner text. This approach
+ * is inspired by WCAG 2.1 Ttechnique G201: Giving users advanced warning when
```

Comment:
```suggestion
 * is inspired by WCAG 2.1 Technique G201: Giving users advanced warning when
```

Reply:
Fixed in https://github.com/hashicorp/dev-portal/pull/1350/commits/c7816ce34214bd25d5c99e63d5227d78722e4042 ✅